tests/bdd/steps/the_fixture.py

```diff
@@ -44,3 +59,64 @@ def load_ini(self, filename, fixture_name):
 
     def get_path(self, filename):
         return os.path.join(self.base_dir, filename)
+
+    def do_teardown(self):
+        self.fixtures = {}
+
+
+@given("fixtures are located in '{location}'")
+def fixture_location(context, location):
+    context.the_fixture.set_base_dir(location)
+
+
+@given("I load the fixture '{fixture_file}.ini' as '{fixture_name}'")
+def load_ini_fixture(context, fixture_file, fixture_name):
+    context.the_fixture.load_ini(fixture_file + ".ini", fixture_name)
+
+
+@given("I set the fixture '{fixture_name}' key '{key}' to '{value}'")
+def set_fixture_value(context, fixture_name, key, value):
+    context.the_fixture.set_fixture_value(fixture_name, key, value)
+
+
+@given("I update the fixture '{fixture_name}' with")
+def update_fixture_from_table(context, fixture_name):
+    for row in context.table:
+        set_fixture_value(context, fixture_name, row[0].strip(), row[1].strip())
+
+
+@then("the fixture '{fixture_name}' key '{key}' is the value")
+def set_fixture_key_to_the_value(context, fixture_name, key):
+    context.the_value = context.the_fixture.get_fixture(fixture_name)[key]
+
+
+def diff_dicts(a, b, missing=KeyError):
+    return {
+        key: (a.get(key, missing), b.get(key, missing))
+        for key in dict(set(a.items()) ^ set(b.items())).keys()
+    }
+
+
+@then("the fixture '{fixture_name}' matches the fixture '{other_fixture}'")
+def compare_fixtures(context, fixture_name, other_fixture):
```

Comment:
I used this during development to test that the fixtures I had created after using the `most_common.ini` and modifying values actually matched the original file for that specific test.